CHANGELOG.rst

```diff
@@ -0,0 +1,47 @@
+Changes from v2.1 to v2.2
+=========================
+
+Deprecated Features
+-------------------
+
+- Deprecated the nominally private class galsim.correlatednoise._BaseCorrelatedNoise.  If you
+  were using it for any purpose, you should now use `galsim.BaseCorrelatedNoise`. (#160)
+- Deprecated the ``tol`` parameter of the various Interpolant classes.  Users should use the
+  ``kvalue_accuracy`` parameter of ``gsparams`` instead. (#1038)
+
+API Changes
+-----------
+
+- Removed functionality to store and re-load WFIRST PSFs, and to obtain multiple WFIRST
+  PSFs simultaneously. (#919)
+- The RealGalaxy COSMOS catalog is now hosted at `Zenodo <https://zenodo.org/record/3242143>`_,
```

Comment:
Is this really an API change?  I thought this should be pretty transparent to any user (shouldn't the COSMOS downloading functionality just work seamlessly without people changing their code?), unlike most API changes.